### ISSUE TITLE
Responsive breakpoint rowheight

### ISSUE THREAD

**Issue by sjhilbel** [miromannino/Justified-Gallery#92]
I would like to see the ability to add a different rowheight for each of the breakpoints I work with. 150px is good for 1200px but not 320px.

**Comment by sjhilbel** [comment]
I was able to get this done with some (window).width if statements.

**Comment by klihelp** [comment]
+1

**Comment by santiquiss** [comment]
+1

**Comment by miromannino** [comment]
Yes, that's can be done with a single call to justified gallery when a resize is done. I prepare you a simple example to put it into the official documentation. Thank you

**Comment by elemis** [comment]
Could you please provide an example on how to add different rowheights for different breakpoints? Thank you

**Comment by bradvin** [comment]
hey @miromannino - have you put this example in the official docs yet? I cannot find it

thanks for any help

**Comment by tomkeysers** [comment]
+1 for the docs update ;)
